docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -781,6 +781,118 @@ To read and write a value of the Bluetooth descriptor:
       - write about setWriteValueRequestCallback() for descriptor
  -->
 ### Responding to read and write value requests from server clients
+
```

Comment:
Please, add a note (somewhere, not necessarily here), that the callbacks described here can also be registered by putting them in `BluetoothGATTServerCharacteristicInit` or `BluetoothGATTServerDescriptorInit`.

Reply:
Please, grep for "desriptor" and replace all occurrences with "descriptor"

Reply:
done